docs/dev_guide/contents/dependencies.rst

```diff
@@ -0,0 +1,22 @@
+.. _dependency_versions:
+
+*************************
+Dependency Support Policy
+*************************
+
+.. note::
+
+    This policy is based on `NEP-0029`_.
+
+sunpy has a short list of core dependencies (Python, numpy, astropy, parfive) and a long list of optional dependencies.
+The minimum version of these packages that we enforce follows this policy.
+
+* Python: Released in the prior 42 months from the anticipated release date.
+* Astropy: Released in the prior 12 months from the anticipated release date.
+* Everything else: Released in the prior 24 months from the anticipated release date.
+
+There are two caveats here: parfive and drms as they are packages we control and might need to bump (especially drms as it's so closely coupled to JSOC).
+
+Sponsored affiliated packages will support the sunpy LTS version at the time of their release.
```

Comment:
Possibly beyond the scope of this PR, but I think we should adopt this for *all* affiliated packages, and have it as a requirement on becoming an affiliated package (ie. you have to support at least one sunpy LTS).

Reply:
I think we should, I think this would go on the sunpy.org as that is where that stuff is hosted.

Unless we have a SEP or something.

Reply:
I am probably not in favour of mandating this for all affiliated packages, but that discussion is for the sunpy.org repo.